### ISSUE TITLE
ValueError: uint8 is not supported

### ISSUE THREAD

**Issue by SimpleXP** [TuSimple/simpledet#164]
Hi,

I have finished the training and try to test the network. With command:
```python3 detection_test.py --config config/tridentnet_r101v2c4_c5_1x.py```
I have the follow output:
```
loading annotations into memory...
Done (t=0.02s)
creating index...
index created!
evaluating [0, 1000)
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/home/xpxu/Project/simpledet/core/detection_input.py", line 740, in collector
    data = [mx.nd.from_numpy(record[name], zero_copy=True) for name in self.data_name]
  File "/home/xpxu/Project/simpledet/core/detection_input.py", line 740, in <listcomp>
    data = [mx.nd.from_numpy(record[name], zero_copy=True) for name in self.data_name]
  File "/usr/local/lib/python3.5/dist-packages/mxnet-1.5.0-py3.5.egg/mxnet/ndarray/ndarray.py", line 4274, in from_numpy
    c_obj = _make_dl_managed_tensor(ndarray)
  File "/usr/local/lib/python3.5/dist-packages/mxnet-1.5.0-py3.5.egg/mxnet/ndarray/ndarray.py", line 4263, in _make_dl_managed_tensor
    c_obj.dl_tensor = _make_dl_tensor(array)
  File "/usr/local/lib/python3.5/dist-packages/mxnet-1.5.0-py3.5.egg/mxnet/ndarray/ndarray.py", line 4250, in _make_dl_tensor
    raise ValueError(str(array.dtype) + " is not supported.")
ValueError: uint8 is not supported.
```
Please advise what shall I do to fix this problem.
Thanks in advance!

**Comment by xchani** [comment]
Seems to be a little problem caused by [mx.nd.from_numpy](https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/ndarray/ndarray.py#L4175-L4187) where ```uint8``` is not support currently. @RogerChern

**Comment by RogerChern** [comment]
Fixed by #166
